CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
   deemed valid instead of invalid ([#650])
 - `[light-client]` Revert a change introduced in [#652] that would enable DoS attacks,
   where full nodes could spam the light client with massive commits (eg. 10k validators).
+- `[tendermint]` (Since RC1) CanonicalBlockId is now correctly decoded to `None` in `CanonicalVote` and `CanonicalProposal`.
```

Comment:
```suggestion
- `[tendermint]` (Since v0.17.0-rc1) CanonicalBlockId is now correctly decoded to `None`
  in `CanonicalVote` and `CanonicalProposal` when its hash is empty.
```

Is this perhaps more explicit about the change?